data/repos.yml

```diff
@@ -327,7 +327,7 @@
 
 - repo_name: govuk-graphql
   team: "#govuk-publishing-platform"
-  alerts_team: "#govuk-publishing-platform-system-alerts"
+  alerts_team: "@richard.towers"
```

Comment:
It appears the use of usernames in place of a channel name has been [deprecated from the Slack API](https://api.slack.com/methods/chat.postMessage#channels).  You'll have to use Richard's user ID instead.

Reply:
My ID is `UAEH5J62J` (I think)

Reply:
```suggestion
  alerts_team: "UAEH5J62J"
```

Reply:
done